### PR TITLE
Added Libraries.io and Dependency CI

### PR DESCRIPTION
**Pull request by andrew**
🚀

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -148,6 +149,7 @@ Table of Contents
   * [saucelabs.com](https://saucelabs.com/) — CI with scalable testing for mobile and web apps, free for Open Source
   * [buddybuild.com](https://www.buddybuild.com/) — Build, deploy and gather feedback for your iOS and Android apps in one seamless, iterative system
   * [gitlab.com](https://about.gitlab.com/gitlab-ci/) - Create pipelines directly from git repositories using GitLab's CI service
+   * [dependencyci.com](https://dependencyci.com/) — Continuous dependency checking for GitHub repositories - free for open source
```
Comment (ripienaar) [inline README.md]: sorry to be a pain, but indent is a tad off :)

**Comment by andrew** [comment]
@ripienaar that's what I get for trying to use the in-browser editor on GitHub!

**Comment by ripienaar** [comment]
perfect thanks, and again congrats on your amazing news @andrew

**Comment by andrew** [comment]
Thanks! It's great to be able to focus on supporting open source communities and pay my bills now!

**Pull request merged**